test/client/fixtures/automation/utils-test.js

```diff
@@ -23,3 +24,20 @@ asyncTest('MoveAutomation._getTargetClientPoint should return integer numbers (G
             start();
         });
 });
+
+asyncTest('Stringify element', function () {
+    const emptyElement             = $('<div />')[0];
+    const elementWithChild         = $('<div><div></div></div>')[0];
+    const elementWithShortText     = $('<div>Short</div>')[0];
+    const elementWithLongText      = $('<div>Long text long</div>')[0];
+    const elementWithChildAndText  = $('<div>Long text long<div></div></div>')[0];
+
+    equal(stringifyElement(null), '');
+    equal(stringifyElement(emptyElement), '<div></div>');
+    equal(stringifyElement(elementWithChild), '<div>...</div>');
+    equal(stringifyElement(elementWithShortText), '<div>Short</div>');
+    equal(stringifyElement(elementWithLongText), '<div>Long te...</div>');
+    equal(stringifyElement(elementWithChildAndText), '<div>...</div>');
+
```

Comment:
Please check more test cases:
* a lot of attributes: `<section id="example-choice-list" class="example-choice-list live" data-property="text-overflow">`
* few nested elements with line separator symbol in their children.